lapce-ui/src/svg.rs

```diff
@@ -49,15 +49,33 @@ pub fn get_svg(name: &'static str) -> Option<Svg> {
 }
 
 pub fn file_svg(path: &Path) -> Svg {
-    let file_type = if path.file_name().and_then(OsStr::to_str) == Some("LICENSE") {
+    
+    // catch if path.to_str() is Some without calling it twice
+    let path_str = path.to_str();
+    if path_str == None {
+        panic!("Missing path");
+    }
+    let file_name = path_str.unwrap(); // unwrap path_str safely now
+
+    let file_type = if file_name.contains("LICENSE") {
         "file_type_license.svg"
+    } else if file_name.to_lowercase().contains("makefile") {
```

Comment:
If you're still willing to work on this PR, please extract the `.to_lowercase()` conversion. In fact, if you could use some sort of case insensitive matching here, that would be best. Thanks :)

Reply:
changed it to .eq_ignore_ascii_case